Image processing/Image compression analyser/utils/file_utils.py

```
@@ -0,0 +1,203 @@
+import os
+import glob
+from pathlib import Path
+from typing import List, Tuple, Optional
```

Ruff (0.12.2)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)